tests/src/rules/named.js

```diff
@@ -135,7 +135,6 @@ ruleTester.run('named', rule, {
 
     test({
       code: 'import { a } from "./re-export-names"',
-      args: [2, 'es6-only'],
```

Comment:
oh actually, I think this line is dead code and can be removed. `es6-only` was a v1 option that was removed ages ago. 

it's fine if it's just `options` now though, can clean up later